### ISSUE TITLE
Expand overwrite and overwritten logic of jumping to correct statement

### ISSUE THREAD

**Issue by AceKuper** [BLCM/OpenBLCMM#44]
Can go to overwritten" and\or go to overwriter, or maybe even better new thing all together, function on logic of go to closest  UP\DOWN. Since now go to overwritten just goes to the closest UP and go to overwriter goes to the latest DOWN. Even then there is a sorta chain of overwrites.

**Comment by apocalyptech** [comment]
Agreed, thx!  The overwriter/overwritten stuff is sort of baked into the mod tree as things are loaded in, so we'd have to change how that's constructed.  Presumably wouldn't be too bad, but I expect this'll probably be for v1.5.0 at the earliest.  Will check it out eventually!
